index.bs

```diff
@@ -715,6 +716,13 @@ A [=sensor=] has an associated <dfn>periodic reporting mode flag</dfn> which is
 
 A [=sensor=] has an associated <dfn>current sampling frequency</dfn> which is initially null.
 
+The [=current sampling frequency=] for a non[=set/is empty|empty=] [=ordered set|set=] of
+[=activated sensor objects=] is an <dfn>optimal sampling frequency</dfn> which is estimated considering
```

Comment:
`estimated considering` reads weird. Maybe a comma before considering or 'which is estimated, taking the ... into account